src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -360,10 +360,30 @@ public override DownloadClientItem GetImportItem(DownloadClientItem item, Downlo
 
         public override DownloadClientInfo GetStatus()
         {
+            var version = Proxy.GetApiVersion(Settings);
             var config = Proxy.GetConfig(Settings);
 
             var destDir = new OsPath(config.SavePath);
 
+            if (Settings.TvCategory.IsNotNullOrWhiteSpace() && version >= Version.Parse("2.0"))
+            {
+                var label = Proxy.GetLabels(Settings)[Settings.TvCategory];
```

Comment:
This could fail is the label is removed from qbit, would be safer to do something like:

```suggestion
                var labels = Proxy.GetLabels(Settings);
                labels.TryGetValue(Settings.TvCategory, out var label);
```

Reply:
Definitely a low risk of this occurring, but rather be safe than sorry.

Reply:
https://github.com/Radarr/Radarr/commit/e7a8f6332c43f179d4c95b3a8a0253a235bb6eec